menu/freedesktop/asebaonlinehelp.desktop

```diff
@@ -7,10 +7,4 @@ Name[es]=Ayuda en línea Aseba
 Name[it]=L'aiuto in linea Aseba
 Name[pt]=ajuda on-line Aseba (Inglês)
 URL=http://www.thymio.org/en:start
-URL[fr]=http://www.thymio.org/fr:start
-URL[de]=http://www.thymio.org/de:start
-URL[es]=http://www.thymio.org/es:start
-URL[it]=http://www.thymio.org/it:start
-URL[pt]=http://www.thymio.org/en:start
-Categories=Education;Science;Aseba
-Icon=asebahelp
\ No newline at end of file
+Icon=asebahelp
```

Comment:
Linter complained about that too. The same section of the spec says that "Categories" is only allowed on Application (i.e. type 1) entries, not Link (i.e. type 3) entries.